### ISSUE TITLE
Disable HTTPS options within SecurePages settings form if HTTPS functionality is not enabled or configured

### ISSUE THREAD

**Issue by jstephendix** [d8-contrib-modules/securepages#6]
Disable the securepages_enable radio form element if https is not available - 
This function in the D7 version checked if the site could be https

`````` /**
 * Secure Pages SSL Test.
 */
function securepages_test() {
  global $is_https;

  // If we are in an SSL page then assume that SSL is configured correctly.
  if ($is_https) {
    return TRUE;
  }
  $url = 'https://' . preg_replace(';^http[s]?://;s', '', url('admin/config/system/securepages/test', array('absolute' => TRUE)));

  $response = drupal_http_request($url);

  return $response->code == 200 ? TRUE : FALSE;
}```
``````

**Comment by jose-oliveira** [comment]
The solution posted on pull request #28, doesn't verify the authenticity of the SSL certificate when making the test call.
Would it be necessary/worth it to allow the user to choose to verify the certificate? (I don't think it is, but I figured I should ask)

**Comment by jstephendix** [comment]
Addressed in PR #28
